config/application.yml.default

```diff
@@ -142,7 +142,6 @@ in_person_enrollments_ready_job_queue_url: ''
 in_person_enrollments_ready_job_max_number_of_messages: 10
 in_person_enrollments_ready_job_visibility_timeout_seconds: 30
 in_person_enrollments_ready_job_wait_time_seconds: 20
-in_person_residential_address_controller_enabled: false
```

Comment:
Do we have to deploy this second after we remove all the places where we read this configuration value?

Reply:
This would be to avoid any issues with the 50/50 state.

Reply:
Oh, heh, great minds think alike! I thought the same thing, but hadn't seen your comment when I started my review.

AIUI, this is actually OK. The new nodes will come up without this config setting, but also without code referencing it. The old nodes, running code that does reference this, will still have the old config. (As evidence of that: we need to do a recycle to pick up config changes.)

But please don't trust this as an inerrant truth. I have been wrong once or twice ~in my career~ so far this morning. ;)

Reply:
In the past we have deleted the config values in the same PR as the rest of the delete code. I see Jack's point and think it is a good suggestion.

Reply:
@n1zyy Yes, but I think the problem is if you bounce between nodes. Really the issue is if you go from a new node without the config value to a new node that expects the config value.

Reply:
is the suggestion then to leave this config value in the pr for now and then remove it in a subsequent pr?

Reply:
@svalexander I think yes, but I think we should do some testing first. Since this is a config value we can toggle it on and off, or set to nil, in a spec. I think what we would want to do would be something like:
1. Set the config value to false or true, this shouldn't matter in new code because we're not reading this value
2. Complete state id step, set same address as id to false
3. Expect to be on the new controller page
4. Set config value to nil
5. Refresh the page
6. Should still be on the new controller page

This will confirm that any transition from an old instance in the 50/50 state to a new instance in the 50/50 state will not crash, as intended. 

I think for the opposite case, where we go from new instance (without config) to old instance that expects a config, will require some testing in the Joy environment following [these steps](https://designsystem.digital.gov/components/overview/).

Reply:
You're not going to get a request from one instance for a config value in another instance. Those calls are all internal to a request. So it's fine to remove the config value within this PR. We did that in all the other delete PRs.

Testing it out as @JackRyan1989 suggests can confirm that.